backend/czi_hosted/data_common/data_adaptor.py

```diff
@@ -158,7 +169,7 @@ def _index_filter_to_mask(self, filter, count):
         mask = np.zeros((count,), dtype=np.bool)
         for i in filter:
             if type(i) == list:
-                mask[i[0]: i[1]] = True
+                mask[i[0] : i[1]] = True
```

Comment:
Asking this question out of curiosity - does the change in this line do anything or is it more of a style thing?

Reply:
Just lint (style, clarity) - no functional change.  We have a programmatic linter and a collection of rules that drive it.  I just ran the linter while I was working on the file.

Reply:
Thank you for the explanation :)